### ISSUE TITLE
hmset() : redisClient.hmset is not a function

### ISSUE THREAD

**Issue by meguoe** [redis/node-redis#2757]
### Description

redisClient.hmset is not a function, Has the hmset function been removed?

**Comment by Smidds** [comment]
Yep! [According to the official Redis docs](https://redis.io/docs/latest/commands/hmset/), as of Redis 4.0.0 HMSET is deprecated and was removed from this project, as explained in #1746 and a few other issues. Thankfully it seems HSET supports setting multiple values, so the functionality remains.
